src/pages/Admin/MapPreview/MapPreview.tsx

```diff
@@ -92,10 +93,10 @@ const MapPreview: React.FC<MapPreviewProps> = ({
     setIsEditing(false);
   };
 
-  const handleBackgroundRemove = () => {
+  const handleBackgroundRemove = async () => {
     if (!user) return;
 
-    return updateVenue_v2(
+    await updateVenue_v2(
```

Comment:
Should the `Promise` still be getting returned here? That's generally how these should work, even if the end result is `Promise<void>`

Reply:
This function is being used by a `div` with `onClick`. Having `return` here miss-leads (at least me) that the result is going to be used but it isn't

Reply:
Fair enough. What is the type on this function at the moment? If it's implicitly `Promise<void>`, then that's fine/nothing required here.

Reply:
Yes, just checked this. It's a void function, no need for the return but the error needs to be handled as you already commented here: https://github.com/sparkletown/sparkle/pull/1229#discussion_r586153454

On it atm.